perfdash/perfdash.go

```diff
@@ -80,12 +85,15 @@ func run() error {
 		options.DefaultBuildsCount = maxBuilds
 	}
 
-	gcs, err := NewGCSMetricsBucket(*logsBucket, *logsPath, *credentialPath)
+	metricsBkt, err := NewGCSMetricsBucket(*logsBucket, *logsPath, *credentialPath)
```

Comment:
enums will make this type of object instantiation nice and clean w/ a switch statement or chain of ifs.